src/plugins/unified_search/public/filter_bar/filter_editor/phrases_values_input.tsx

```diff
@@ -40,11 +41,14 @@ class PhrasesValuesInputUI extends PhraseSuggestorUI<Props> {
             id: 'unifiedSearch.filter.filterEditor.valuesSelectPlaceholder',
             defaultMessage: 'Select values',
           })}
+          delimiter=","
           options={options}
           getLabel={(option) => option}
           selectedOptions={values || []}
           onSearchChange={this.onSearchChange}
-          onCreateOption={(option: string) => onChange([...(values || []), option])}
+          onCreateOption={(option: string) => {
+            this.props.onParamsUpdate(option.trim());
+          }}
```

Comment:
just curious, was there an issue that prevented us from reusing `onChange` here?

also a consistency nit if we need to keep it this way: since we are already destructuring `this.props` can we do:
```
onParamsUpdate(option.trim());
```

Reply:
Thank you so mych @jloleysens for your review. Yes I tried to use the onChange but it didn't work correctly. There was a sync problem, it didn't take under consideration the existing state. I saw that on the EUI documentation they are also proposing to go with the prevState https://elastic.github.io/eui/#/forms/combo-box#with-delimiter

I will address your nit!